source/wifi/wifi_hal.c

```diff
@@ -8118,6 +8183,11 @@ int main(int argc,char **argv)
         return wifi_reset();
     }    
 
+    if(argc <= 2)
```

Comment:
can you please add (argc <= 2) in first "if" condition of main function and remove this 5 lines.

Reply:
I cannot just add (argc <= 2) in first "if" because the "if-else" that is above will not work. I will move first "if" condition below and then it will work.

Reply:
Point taken.